src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Interfaces/IDiscountRepository.cs

```diff
@@ -11,36 +11,36 @@ public interface IDiscountRepository
     {
 
         /// <summary>
-        /// Gets Discount Programs by SearchCriteria.
+        /// Gets Discounts by SearchCriteria.
         /// </summary>
         /// <returns></returns>
         Task<List<DiscountResponse>> GetDiscountsAsync(SearchCriteria searchCriteria);
 
         /// <summary>
-        /// Gets a Discount Program by id.
+        /// Gets a Discount by id.
         /// </summary>
-        /// <param name="id">The id Discount Program.</param>
+        /// <param name="id">The id Discount.</param>
         /// <returns></returns>
         Task<DiscountResponse> GetDiscountAsync(Guid id);
```

Comment:
GetDiscountByUidAsync for example

Reply:
[Done](https://github.com/yegorl/exadel_discounts_be/blob/CrazyPrice_007_Controllers/src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Interfaces/IDiscountRepository.cs)